### DISCUSSION TITLE
[Commission] A Simple Counter Plugin

### DISCUSSION THREAD

**Discussion by AdequateAce** [runelite/runelite#16976] [Ideas]
I just obtained the Infinite Coin Bag on my snowflake ironman. I figure it'll be fun AND beneficial to pull coins from this thing any time I have downtime (agility, between attacks in combat, questing).

I want a counter that logs how many coins I have pulled from the bag total, and, if possible, displays the number on top of the icon similar to any stackable item's quantity.

If you are interested, please message me on Discord (adequateace) and we can figure out the details/pay!

Thank you!

**Comment by AdequateAce** [comment]
Bump

**Comment by rugg0064** [comment]
Sent you a DM on Discord - rugg0064
